testes-api/cypress/support/commands.js

```diff
@@ -174,4 +177,80 @@ Cypress.Commands.add('adminCreatesAMovie', (title, genre, description, durationI
       }, failOnStatusCode
     })
   })
+});
+
+Cypress.Commands.add('getUserViaID', (id, failOnStatusCode) => {
+  cy.request({
+    method: "GET",
+    url: "users/" + id,
+    headers: {
+      Authorization: `Bearer ${Cypress.env('accessToken')}`
+    },
+    failOnStatusCode: failOnStatusCode
+  })
+})
+
+Cypress.Commands.add('createMovie', () => {
+  let idFilme
+  cy.request({
+    method: "POST",
+    url: "movies",
+    headers: {
+      Authorization: `Bearer ${Cypress.env('accessToken')}`
+    },
+    body: {
+      title: faker.person.firstName() + " o retorno",
+      genre: "Terror",
+      description: faker.lorem.words({ min: 6, max: 12 }),
+      durationInMinutes: 120,
+      releaseYear: 2000,
+    },
+  }).then((response) => {
+    idFilme = response.body.id
+    return { idFilme: idFilme }
```

Comment:
```suggestion
    return response.body
``` 
e excluir linha 209 - assim eu já aproveito pra validar no meu search :D